src/lib_ccx/params.c

```diff
@@ -34,47 +34,58 @@
 
 static int inputfile_capacity=0;
 
-int process_cap_file (char *filename)
+#define MAX_WORD_LENGTH 50
```

Comment:
Keep in mind that for now at least we're only doing this for 608 subtitles (as far as I can see), and the width of a line in 608 is 32...

Reply:
Ok, I thought I was set arbitrarily.